manim/scene/moving_camera_scene.py

```diff
@@ -83,8 +102,10 @@ class MovingCameraScene(Scene):
     This is a Scene, with special configurations and properties that
     make it suitable for cases where the camera must be moved around.
 
+    Note: Examples are included in the moving_camera_scene module
+    documentation, see below in the 'see also' section.
     .. SEEALSO::
-
+        :mod:`.moving_camera_scene`
```

Comment:
This renders incorrectly, we might need an extra line before and after the `.. SEEALSO::`

Reply:
Thanks, I have added the change

Reply:
Thank you! We have added extra line before and after `.. SEEALSO::`. 
Is it okay now?